tox.ini

```diff
@@ -23,4 +23,5 @@ exclude =
     __pycache__
 ignore =
     W503
+    RST304 # Unknown interpreted text role
```

Comment:
Maybe this ignore should actually not be added. The warning it resulted in may have been correct and would have been picked out more accurately by https://github.com/ComPWA/tensorwaves/pull/27.

Reply:
yeah, better we add a custom defined role instead of ignoring all

Reply:
https://github.com/ComPWA/tensorwaves/pull/30/commits/063b260e45411f101b567406bbe92f07e5ca3bd1